metabulo/views.py

```diff
@@ -96,6 +97,22 @@ def delete_csv_file(csv_id):
 
     return '', 204
 
+# Ingest transforms
+@csv_bp.route('/csv/<uuid:csv_id>/normalization', methods=['PUT'])
+def set_normalization_method(csv_id):
+    csv_file = CSVFile.query.get_or_404(csv_id)
+    args = request.json
+    if args is not None and args not in NORMALIZATION_METHODS:
```

Comment:
Could we change to key-value pair, like `{ type: "minmax" }` or something?  Axios tries to get clever and sends mime type based on data payload, which results in `minmax` being sent as formdata.